checker/values.ml

```diff
@@ -236,7 +236,7 @@ let v_template_universes =
   v_tuple "template_universes" [|List(Opt v_level);v_context_set|]
 
 let v_primitive =
-  v_enum "primitive" 50 (* Number of "Primitive" in Int63.v and PrimFloat.v *)
+  v_enum "primitive" 54 (* Number of "Primitive" in Int63.v and PrimFloat.v *)
```

Comment:
In passing, the comment can probably be improved as something along the line "number of constructors of the CPrimitives.t type".